partiql-planner/src/test/kotlin/org/partiql/planner/PlannerErrorReportingTests.kt

```diff
@@ -264,28 +247,30 @@ internal class PlannerErrorReportingTests {
                 "not_a_function(1)",
                 false,
                 assertOnProblemCount(0, 1),
-                StaticType.MISSING
+                StaticType.ANY
             ),
             TestCase(
                 "not_a_function(1)",
                 true,
                 assertOnProblemCount(0, 1),
-                StaticType.MISSING
+                StaticType.ANY
             ),
 
             // 1 + not_a_function(1)
             //  The continuation will return all numeric type
+            // TODO: Should the warning count be 1? Does it matter if it is zero?
```

Comment:
Probably shouldn't matter that the warning count is 0 if an error is already being reported.